packages/react-scripts/template/src/index.css

```diff
@@ -1,5 +1,12 @@
+* {
+  box-sizing: border-box;
```

Comment:
I agree with this one. But maybe we should just bring in some good CSS reset by default? Can you suggest any?